dll/win32/comctl32/propsheet.c

```diff
@@ -3580,6 +3580,28 @@ PROPSHEET_DialogProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
         }
         SetFocus(GetDlgItem(hwnd, IDOK));
       }
+#ifdef __REACTOS__
+      { /* try to fit it into the desktop */
+          RECT rcWork;
+          RECT rcDlg;
+          int dx, dy;
+
+          if (GetWindowRect(hwnd, &rcDlg) && SystemParametersInfo(SPI_GETWORKAREA, 0, &rcWork, 0))
+          {
+              dx = rcDlg.right - rcWork.right;
+              dy = rcDlg.bottom - rcWork.bottom;
+
+              if (dx > 0)
+                  rcDlg.left -= dx;
+              if (dy > 0)
+                  rcDlg.top -= dy;
+              if (rcDlg.right > rcWork.right)
+                  rcDlg.right = rcWork.right;
```

Comment:
NOTE: This two lines are not effective.

Reply:
@katahiromz  I agree with you, but as you can see it was a request from @ThFabba . What should I do now?

Reply:
That must have been a miscommunication. I never intended to ask you to add code that does nothing.
What I was trying to get at: `rcDlg.right > rcWork.right` should be used instead of `dx > 0` because the subtraction that calculated `dx` may have overflowed.

Reply:
The correct test is dx > 0 and dy > 0.  To prevent overflow